view_controllers/OBARegionListViewController.m

```diff
@@ -343,7 +343,13 @@ - (void)showLocationServicesAlert {
         UIAlertView *view = [[UIAlertView alloc] init];
         view.title = NSLocalizedString(@"Location Services Disabled", @"view.title");
         view.message = NSLocalizedString(@"Location Services are disabled for this app. Some location-aware functionality will be missing.", @"view.message");
-        [view addButtonWithTitle:NSLocalizedString(@"Dismiss", @"view addButtonWithTitle")];
+        view.delegate = self;
+        [view addButtonWithTitle:NSLocalizedString(@"Okay", @"Ok button")];
+
+        if (&UIApplicationOpenSettingsURLString != NULL) {
+            [view addButtonWithTitle:@"Location Settings"];
```

Comment:
Where can I read to understand the purpose of the second argument of `NSLocalizedString`? What is the best way to tell which button was pressed other than my hacky method of button title?